src/index.html

```diff
@@ -2,16 +2,128 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>
+          Surname: <input type="text"
+                          name="surname"
+                          required
+                          autocomplete="off"
+                   >
```

Comment:
```suggestion
          Surname:
          <input
             type="text"
             name="surname"
             required
             autocomplete="off"
           >
```

check your code with [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)